eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json

```diff
@@ -0,0 +1,687 @@
+{
+  "process_graph": {
+    "1": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "31"
+        },
+        "cube1": {
+          "from_node": "10"
+        }
+      }
+    },
+    "2": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "30"
+        },
+        "cube1": {
+          "from_node": "11"
+        }
+      }
+    },
+    "3": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "36"
+        },
+        "cube1": {
+          "from_node": "34"
+        }
+      }
+    },
+    "4": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "25"
+        },
+        "cube1": {
+          "from_node": "35"
+        }
+      }
+    },
+    "5": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube1": {
+          "from_node": "40"
+        },
+        "cube2": {
+          "from_node": "41"
+        }
+      }
+    },
+    "6": {
+      "process_id": "load_collection",
+      "arguments": {
+        "temporal_extent": [
+          "2015-11-06T00:00:00.000Z",
+          "2016-09-27T00:00:00.000Z"
+        ],
+        "spatial_extent": {
+          "west": 10.570392608642578,
+          "south": 46.78148963659169,
+          "east": 10.777416229248047,
+          "north": 46.85244345762143
+        },
+        "id": "Backscatter_Sentinel1_Track015"
+      }
+    },
+    "9": {
+      "process_id": "load_collection",
+      "arguments": {
+        "temporal_extent": [
+          "2014-12-04T00:00:00.000Z",
+          "2014-12-08T00:00:00.000Z"
+        ],
+        "spatial_extent": {
+          "west": 10.570392608642578,
+          "south": 46.78148963659169,
+          "east": 10.777416229248047,
+          "north": 46.85244345762143
+        },
+        "id": "LIA_Sentinel1_Track015",
+        "bands": [
+          "LIA"
+        ]
+      }
+    },
+    "99": {
+      "process_id": "reduce_dimension",
+      "arguments": {
+        "data": {
+          "from_node": "9"
+        },
+        "reducer": {
+          "process_graph": {
+            "1": {
+              "process_id": "min",
+              "arguments": {
+                "data": {
+                  "from_parameter": "data"
+                }
+              },
+              "result": true
+            }
+          },
+          "parameters": [
```

Comment:
All these "empty" parameters should be removed.

Reply:
see also #10

Reply:
I fixed it, now should be fine.